post-type-archive-descriptions.php

```diff
@@ -44,6 +44,31 @@ function ptad_get_post_types() {
 	return $post_types;
 }
 
+/**
+ * Output filterable parent of Settings page
+ *
+ * @param string $post_type name of post type for the page
+ * @return bool|string parent for settings page
+ */
+function ptad_settings_page_parent( $post_type, $show_in_menu = false ) {
```

Comment:
`$show_in_menu` needs documentation. The name is a little odd in this context even though I realize it's coming straight from the WP naming conventions. It's tempting to give it a different name, but that might just confuse things further.